### PR TITLE
EWPP-129: Add site information service and Behat context

### PR DESCRIPTION
**Pull request by ademarco**

### CODE REVIEW

File: tests/src/Behat/CorporateSiteInformationContext.php [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  /**
+   * Load a SKOS concept entity given its label, if any.
+   *
+   * @param string $label
+   *   Entity label.
+   *
+   * @return \Drupal\rdf_skos\Entity\Concept
+   *   SKOS entity object.
+   */
+  protected function loadEntityByLabel(string $label): Concept {
```
Comment (imanoleguskiza) [inline tests/src/Behat/CorporateSiteInformationContext.php:81]: We should either rename this method to loadSkosConceptByLabel or pass a parameter (entity_type) for storage.

File: tests/features/corporate-information.feature [hunk 2]
```diff
@@ -0,0 +1,9 @@
+@api
+Feature: Corporate information
+  In order to be able to set and assert corporate information in my Behat tests
+  As a developer
+  I want to make sure that the steps provided by this module work correctly.
+
+  Scenario: Set and assert content owner.
+    Given I set the site owner to "Directorate-General for Informatics"
```
Comment (imanoleguskiza) [inline tests/features/corporate-information.feature:8]: This test would make more sense as a kernel test right? A behat test does not seem to offer any value to the PrOw. Unless we have this test as a showcase of the custom step, in which case its fine.

**Review by imanoleguskiza — CHANGES_REQUESTED** [review]

File: tests/features/corporate-information.feature (hunk 2 above)
Comment (ademarco) [reply tests/features/corporate-information.feature:8]: Yeah it's only to test the steps themselves, as i say in the feature secription: As a developer...

File: src/SiteInformation.php [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+   * {@inheritdoc}
+   */
+  public function hasSiteOwner(): bool {
+    return (bool) $this->configFactory->get(self::CONFIG_NAME)->get('site_owner');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getSiteOwner(): Concept {
```
Comment (sergepavle) [inline src/SiteInformation.php:58]: ConceptInterface

File: src/SiteInformationInterface.php [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+   */
+  public function hasSiteOwner(): bool;
+
+  /**
+   * Get current site owner SKOS concept entity.
+   *
+   * @return \Drupal\rdf_skos\Entity\Concept
+   *   The current site owner SKOS concept entity.
+   */
+  public function getSiteOwner(): Concept;
```
Comment (sergepavle) [inline src/SiteInformationInterface.php:28]: ConceptInterface

File: tests/src/Behat/CorporateSiteInformationContext.php [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+    $this->configContext->setConfig('oe_corporate_site_info.settings', 'site_owner', $entity->id());
+  }
+
+  /**
+   * Assert site owner.
+   *
+   * @param string $label
+   *   Site owner SKOS concept label.
+   *
+   * @Then the site owner should be set to :arg1
```
Comment (sergepavle) [inline tests/src/Behat/CorporateSiteInformationContext.php:56]:
```suggestion
   * @Then the site owner should be set to :label
```

**Review by sergepavle — CHANGES_REQUESTED** [review]

**Review by sergepavle — APPROVED** [review]

**Comment by ademarco** [comment]
@imanoleguskiza @sergepavle I'll take over this again, we forgot to add the concept subset to the site owner:
```
      'concept_subset' => 'oe_corporate_site_info_corporate_bodies_department_executive_agencies',
```

**Review by 22Alexandra — APPROVED** [review]

**Review by imanoleguskiza — APPROVED** [review]

**Pull request merged**